### ISSUE TITLE
Tabindex attribute should not be normally needed

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-accessibility-passport#115]
From AAACT:

> Tabindex attribute is used to insert an element to the tab order that is not in the natural tab order or to remove an element that is. Please note that the tabindex attribute should not be normally needed when the HTML5 specifications are correctly followed. The incorrect usage of the tabindex attribute may cause a failure under this criterion. Currently, there is one tabindex=”-1” on a DIV which does nothing, as DIVs are not in the tab order anyway.

**Comment by mnigh** [comment]
@yonikid15 was there a specific reason for the [tabindex on this div](https://github.com/GCTC-NTGC/gc-accessibility-passport/commit/5a75f9e7d159ee8ebc6648956e136f50781ccd58#diff-0348808fef58c5568d86baa15ff97eeb8e2990127b9ffbaeb0ee7b59324d8721R44)?

**Comment by yonikid15** [comment]
> @yonikid15 was there a specific reason for the [tabindex on this div](https://github.com/GCTC-NTGC/gc-accessibility-passport/commit/5a75f9e7d159ee8ebc6648956e136f50781ccd58#diff-0348808fef58c5568d86baa15ff97eeb8e2990127b9ffbaeb0ee7b59324d8721R44)?

No that must be a mistake.
